ROOTFS/usr/bin/waggle_sanity_check_interactive.py

```diff
@@ -59,18 +59,21 @@ def execute_tests_in_path(tests_dir, tests_severity):
     testsFailed = []
 
     logging.info(f"Executing Tests in Dir: {tests_dir}")
-    for filename in os.listdir(tests_dir):
+    tests = os.listdir(tests_dir)
```

Comment:
Conceptually looks good overall. I think you could probably streamline a lot of this by leveraging Path in pathlib. For example, something like:

```python
import subprocess
from pathlib import Path

for path in sorted(Path("sanity/warning").glob("*.test")):
    print("running", path.stem) # .stem for "sanity/warning/factory_prov.test" -> factory_prov
    subprocess.run(path)
    # ... rest of code ...
```

Reply:
like it. done